helpers/random_values.ts

```diff
@@ -14,3 +14,9 @@ export function getStringWithSpaceInEnd() {
     return `${part1}${part2} `
 }
 
+export function getRandomLetter() {
```

Comment:
You can replace it with these functions. And then use the `getRndStr(1)` to get random letter
```ts
const characters = {
    letters: 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz',
    numbers: '0123456789',
};

function getRnd(length: number, characters: string): string {
    let result = '';
    const charactersLength = characters.length;
    for (let i = 0; i < length; i++) {
        result += characters.charAt(Math.floor(Math.random() * charactersLength));
    }
    return result;
}

/**
 * Get random string only with letters.
 *
 * @param length - length of the string.
 * @returns a string with a random number of letter
 */
export function getRndStr(length: number): string {
    return getRnd(length, characters.letters);
}

/**
 * Get random string only with numbers.
 *
 * @param length - length of the string.
 * @returns a string with a random number of numbers
 */
export function getRndStrNumbers(length: number): string {
    return getRnd(length, characters.numbers);
}
```

Reply:
This function is used only once. So I would rather leave it as it is